demo/docker-compose.yaml

```diff
@@ -1,5 +1,46 @@
 version: "3.9"
 services:
+  spark-master:
```

Comment:
We can run spark from the notebook currently, should we just do that?

Reply:
Leaving this PR open until I could confirm quickstart examples on the demo jupyter notebook works seamlessly.